app/models/talk.rb

```diff
@@ -49,14 +49,16 @@ class Talk < ApplicationRecord
     attribute :speakers do
       speakers.pluck(:name)
     end
-    searchable_attributes [:title, :description]
+    add_attribute :_vectors do
+      vectors
+    end
+    searchable_attributes [:title, :description, :_vector]
```

Comment:
I am pretty sure you don't want to let users search into a list of floats 😄

Reply:
Yes indeed :), a leftover from some previous trial and error, thanks for the catch!